sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -401,7 +403,7 @@ def _get_name_or_component_name(node: Union[BaseNode, AutoMLJob]):
 
 
 def _build_pipeline_parameter(
-        func, *, user_provided_kwargs, group_default_kwargs=None, non_pipeline_parameter_dict=None):
+        func, *, user_provided_kwargs, group_default_kwargs=None, non_pipeline_parameters=None):
```

Comment:
maybe call this non_pipeline_inputs?

Reply:
rename non_pipeline_parameters to non_pipeline_inputs